amida-tech/greyscale
# Bug/inba 752 po notify users new task project

Sends out email / internal notification when a user is assigned a new task and when a users task is now active.

Ticket: https://jira.amida-tech.com/browse/INBA-752

To Test:
1.
- Create a project (or use an existing project)
- Assign a new task to a user with a legit email that you can check
- Check to make sure that user got an email and also an internal notification.

2.
- Assign a task to user A in stage 1
- Assign a task to user B in stage 2
- Complete the task for user A in stage 1
- Ensure that user B gets an email and internal notification saying their task is now active

The email being sent out for task activation could probably contain some more information- but this took unnecessarily long to get done and it seems like to do that will require some pretty serious modifications- so I would rather separate that into a different ticket:
https://jira.amida-tech.com/browse/INBA-757
